app/components/listing/Services.jsx

```diff
@@ -80,7 +80,8 @@ updated
                 result={service.fee}
               />
               {service.notes.length ? <Notes notes={service.notes} /> : null }
-              <WeeklyHours schedule={service.schedule} />
+              {service.recurringSchedule.intervals.length > 0
```

Comment:
is it possible that recurringSchedule won't have intervals? Might want to guard against that either here by using lodash get, or in the RecurringSchedule class by setting it to an empty array if intervals aren't passed into the constructor.

Reply:
Good idea, added that in 3303699.